polychromatic-tray-applet

```diff
@@ -440,6 +440,7 @@ class AppIndicator(object):
             self.indicator.set_menu(root)
```

Comment:
in that if branch maybe the `self.indicator.set_title("Polychromatic")` is missing, as seen on line 387

Reply:
Not entirely sure if `set_title` is absolutely necessary generally, but it would be good to add it here for consistency :+1: